### DISCUSSION TITLE
Welcome to pantab Discussions!

### DISCUSSION THREAD

**Discussion by WillAyd** [innobi/pantab#262] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Carlifornia-Five** [comment]
Hi,
  In the topic of appending data the documentation refers to:
   
   new_data = pd.DataFrame([["moose", 4]], columns=["animal", "num_of_legs"])

   # Instead of overwriting the animals table, we can append via table_mode
    pt.frame_to_hyper(df, "example.hyper", table="animals", table_mode="a")
    
   It should say:
   
   new_data = pd.DataFrame([["moose", 4]], columns=["animal", "num_of_legs"])

   # Instead of overwriting the animals table, we can append via table_mode
    pt.frame_to_hyper(new_data, "example.hyper", table="animals", table_mode="a")

**Reply by WillAyd** [reply]
Hi @Carlifornia-Five - if you see an issue with the documentation feel free to put up a pull request to change it - edits are always welcome!
